core/src/scene/sceneLoader.cpp

```diff
@@ -66,6 +73,261 @@ glm::vec3 parseVec3(const Node& node) {
     return vec;
 }
 
+void SceneLoader::loadShaderConfig(YAML::Node shaders, ShaderProgram& shader) {
+
+    if (!shaders) {
+        return;
+    }
+
+    Node definesNode = shaders["defines"];
+    if (definesNode) {
+        for (const auto& define : definesNode) {
+            std::string name = define.first.as<std::string>();
+            std::string value = define.second.as<std::string>();
+            shader.addSourceBlock("defines", "#define " + name + " " + value);
```

Comment:
Gotta add special treatment for booleans here, the JS code uses booleans to define existence: https://github.com/tangrams/tangram/blob/73657bd3c85eb68605398b2735eaa69b3c79126e/src/gl/shader_program.js#L410-L415

@bcamper if one wanted to #define an actual boolean, do 1 and 0 evaluate correctly?